docs/processes/coding standard/Wordpress/WPCS.md

```diff
@@ -0,0 +1,178 @@
+  3. As this function are not wordpress specific. It will give error for acf methods like get_field, have_rows.
+  4. For removing the above error clone this [acf-pro-stubs](https://github.com/php-stubs/acf-pro-stubs) in your document directory
+  5. Go to Vscode setting in JSON format add the acf-pro-stubs directory for PHP Intelephense
+
+       ```array
+        "intelephense.stubs": [ 
+          “WordPress”,
+          ”/var/www/html/stubs/acf-pro-stubs”,
+        ]
+        ```
+  - Now the ACF methods error will be gone.
+
+## Important Extension for Development
+
+  - Using [acf-snippets](https://marketplace.visualstudio.com/items?itemName=anthonydiametrix.ACF-Snippet) vscode extension by *Anthony Hubble*
+    1. field:if / field:ifelse
+        ```php
+        <?php if ( get_field('field_name') ) : ?>
+        <?php endif; ?>
+        ```
+    2. field:sub
+        ```php
+          <?php if ( get_sub_field('field_name') ) : ?>
+            <?php echo get_sub_field('field_name'); ?>
+          <?php endif; ?>
+        ```
+    3. field:image
+        ```php
+          <?php if ( get_field('field_name') ) : ?>
+            <img src="<?php the_field('field_name'); ?>" alt="<?php the_field(''); ?>">
+          <?php endif; ?>
+        ```
+    4. field:flex *(Flexible Content Field)*
+        ```php
+          <?php if ( have_rows( 'field_name' ) ) : ?>
+            <?php while ( have_rows('field_name' ) ) : the_row();
+              if ( get_row_layout() == 'layout_field' ) : ?>
+                <div class="class">
+                  <?php the_sub_field( 'field_name' ); ?>
+                </div>
+              <?php endif; ?>
+            <?php endwhile; ?>
+          <?php endif; ?>
+        ```
+    5. field:relationship *(Relationship Field)*
+        ```php
+          <?php $posts = get_field('field_name'); ?>
+          <?php if ( $posts ): ?>
+            <ul>
+              <?php foreach ( $posts as $post ) : setup_postdata( $post ); ?>
+                <li>
+                  <a href="<?php the_permalink(); ?>"><?php the_title(); ?></a>
+                </li>
+              <?php endforeach; wp_reset_postdata(); ?>
+            </ul>
+          <?php endif; ?>
+        ```
+    6. For more snippets checkout this [doc](https://docs.google.com/document/d/1CTLZ6f2agwWT4IiI7ZTEoQ0OrrZBDxDZxCHltRZrwG8/edit).
+  - Use [WordPress & ClassicPress snippets plus autocompletions](https://marketplace.visualstudio.com/items?itemName=laurencebahiirwa.classicpress-snippets) by *Laurence Bahiirwa*: For Wordpress Method autocompletions
+
+## Fixing Minor formatting issues
+
+1. For fixing Some issues in a file use this command `phpcbf --standard="WordPress"  header.php` *( This command will format your WP php file )*
+2. To Fix some minor issue in all php file in a project use this command: `phpcbf -p -s -v -n . --standard=WordPress --extensions=php`
+
```

Comment:
reporting option is missing can you please add some details around it as well 
format how we can create error reports